### DISCUSSION TITLE
Neper crystal orientation

### DISCUSSION THREAD

**Discussion by joshuajaistein** [neperfepx/neper#404] [Q&A] [answered]
Is there a way to give specific orientation to the cell arrangement structure in neper ?
![VORONOI-PATTERN 1](https://user-images.githubusercontent.com/55456301/157203558-85f9d789-d268-41c3-ab20-518c66eaba7b.png)

**Comment by rquey** [comment, marked as answer]
If you mean elongated cells, then it can be done using [-morpho aspratio](https://neper.info/doc/neper_t.html#cmdoption-morpho):
```
neper -T -n 1000 -morpho "aspratio(2,1,0.5)"
neper -V n1000-id1.tess -print img
```
![image](https://user-images.githubusercontent.com/2659376/157211416-1c79bd90-4e11-4c0b-b1a9-9ccc34e24557.png)
